### PR TITLE
Minor correction in test_searchlight.py

### PR DESCRIPTION
**Pull request by SylvainTakerkart**
Corrected the definition of the conditions (targets / y) in the searchlight test function so that the classes are balanced

### CODE REVIEW

File: nilearn/decoding/tests/test_searchlight.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def test_searchlight():
     mask = np.ones((5, 5, 5), np.bool)
     mask_img = nibabel.Nifti1Image(mask.astype(np.int), np.eye(4))
     # Create a condition array
```
Comment (GaelVaroquaux) [inline nilearn/decoding/tests/test_searchlight.py]:
```suggestion
    # Create a condition array, with balanced classes
```

**Comment by kchawla-pi** [comment]
Link to sphere_mesh: https://osf.io/vym9k/download

**Comment by GaelVaroquaux** [comment]
LGTM once travis has run.

**Comment by GaelVaroquaux** [comment]
Tests have run. Merging.

**Pull request merged**